lisa/messages.py

```diff
@@ -38,6 +38,19 @@ class TestRunMessage(MessageBase):
 @dataclass
 class PerfMessage(MessageBase):
     type: str = "Performance"
+    test_case_name: str = ""
```

Comment:
Because every message needs to initialize those fields, add a method in this module to share like below. Refer to other place like subclasses.py to use generic type.

```python
create_message(message_type: Type[T], node:Node) -> T:
    message = message_type()
    ....
    return T
```

Reply:
> Because every message needs to initialize those fields, add a method in this module to share like below. Refer to other place like subclasses.py to use generic type.
> 
> ```python
> create_message(message_type: Type[T], node:Node) -> T:
>     message = message_type()
>     ....
>     return T
> ```

I am not quite sure about this, could you please take a look at https://github.com/microsoft/lisa/pull/1698/files#diff-aebab7e1684e26efc5eeac6fd09107ce17f2ec4aa641cd40baa10c81c96d1b0fR177?

Reply:
replied in another comment